cmd/kubeadm/app/util/logfile.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+// kubeadm record file path and name define
+const (
+	KubeadmLogFolder = "/var/lib/kubeadm"
+	KubeadmLogFile   = KubeadmLogFolder + "/" + "kubeadm.log"
+)
+
+// WriteKubeadmLogFile write record to kubeadm log file when kubeadm init or kubeadm join
+func WriteKubeadmLogFile(content string) error {
```

Comment:
to be able to add unit tests to we need to separate the public function and the implementation function.

let's do this:
```
func WriteKubeadmLogFile(content string) error {
	return writeKubeadmLogFile(kubeadmLogFolder, kubeadmLogFile, content)
}

func writeKubeadmLogFile(folder, file, content string) error {
	....
	if err := ioutil.WriteFile(path.Join(folder, file), ...) {
	....
}
```